src/settings.c

```diff
@@ -91,6 +91,12 @@ static void on_symbols_tree_expanders_visible_changed(GSettings *settings, gchar
 }
 
 
+static void on_notebook_double_click_hides_widgets_changed(GSettings *settings, gchar *key, gpointer user_data)
+{
+	interface_prefs.notebook_double_click_hides_widgets = g_settings_get_boolean(settings, key);
```

Comment:
@elextr ones like this (which are just for backwards compat BTW) should be easily made generic/reused in a manner similar to `on_tab_pos_changed` above.